build/azure-pipelines/build-product.yml

```diff
@@ -168,4 +168,13 @@ stages:
         inputs:
           targetPath: $(Build.ArtifactStagingDirectory)
           artifactName: SqlcmdRelease
-
+      - task: GitHubRelease@1
+        inputs:
+          gitHubConnection: 'gosqlcmd_github'
+          repositoryName: '$(Build.Repository.Name)'
+          action: 'create'
+          target: '$(Build.SourceVersion)'
+          tagSource: 'gitTag'
+          tagPattern: 'v*.*.*'

```

Comment:
> tagPattern: 'v*.*.*' [](http://example.com/codeflow?start=9&length=21)

we already know the actual tag, it's $(VERSION_TAG)

Reply:
Updated the tag.